.github/workflows/test-amazon-web-services-install.yaml

```diff
@@ -9,6 +9,17 @@ name: e2e - Amazon Web Services (install)
 
 on:
   workflow_dispatch:
+    inputs:
+      posthog-image-sha:
+        description: 'The posthog docker image to deploy'
+        required: false
+        type: string
+      keep-after-install:
+        description: ''
+        required: true
+        default: "false"
```

Comment:
That's not a boolean?

Reply:
Hah yeah haven’t looked at this one yet, for some reason my actions linting wasn’t happy with an actual boolean

Reply:
Note to self, perhaps related: https://github.com/actions/runner/issues/1483

Will try to get this PR through on Monday and fix remaining issues